it/src/test/java/com/linecorp/armeria/server/grpc/interop/ArmeriaGrpcServerInteropTest.java

```diff
@@ -128,46 +112,50 @@ public void clearCtxCapture() {
     @Override
     protected ManagedChannel createChannel() {
         try {
-            // Use reflection to access package-private method.
-            Method getPort = AbstractInteropTest.class.getDeclaredMethod("getPort");
-            getPort.setAccessible(true);
-            return NettyChannelBuilder
-                    .forAddress("localhost", (int) getPort.invoke(this))
-                    .flowControlWindow(65 * 1024)
+            final int port = getPortViaReflection();
+            return OkHttpChannelBuilder
```

Comment:
AFAICT, the client doesn't support configuring it, it will just use what's in the HTTP/2 Settings frame with a fallback.

https://github.com/grpc/grpc-java/blob/master/okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java#L958